apps/builder/app/builder/features/sidebar-left/panels/marketplace/utils.ts

```diff
@@ -0,0 +1,136 @@
+import { useStore } from "@nanostores/react";
+import { atom } from "nanostores";
+import { nanoid } from "nanoid";
+import type { MarketplaceProduct } from "./types";
+import { loadProjectDataById, type Data } from "@webstudio-is/http-client";
+import {
+  extractWebstudioFragment,
+  findTargetAndInserFragment,
+} from "~/shared/instance-utils";
+import type { WebstudioData } from "@webstudio-is/sdk";
+
+const $activeProductId = atom<MarketplaceProduct["id"] | undefined>();
+
+export const $activeProductData = atom<WebstudioData | undefined>();
+
+export const products: Array<MarketplaceProduct> = [
+  // @todo use the right product, find a better place for this initial list
```

Comment:
@TrySound thoughts on this? since we aren't gonna make it through the API right now, should we try and find a better place than this?

We also can't really use env because then we have to inline a json and that's a bit ugly and error prone